dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftMetadataConnector.java

```diff
@@ -64,6 +93,10 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
     out.add(new DumpMetadataTask(arguments, FORMAT_NAME));
     out.add(new FormatTask(FORMAT_NAME));
     out.add(new RedshiftEnvironmentYamlTask());
+    // AWS API tasks
+    if (arguments.getDefinition(RedshiftMetadataConnectorProperties.AWS_API) != null) {
+      out.add(new RedshiftClusterNodesTask());
```

Comment:
Why don't we do this by default, if we have the right credentials?

Reply:
I changed the credentials flow based on your suggestions.